frontend/sequences/step_tiles/__tests__/tile_move_absolute_test.tsx

```diff
@@ -80,6 +80,26 @@ describe("<TileMoveAbsolute/>", () => {
     expect(mock.mock.calls[0][0].location.args.x).toBe(23.456);
   });
 
+  it("Options visible on greater screen width", () => {
```

Comment:
In future tests, please use a lowercase verb as the first word in an `it("")` statement.

Example:

```
it("renders visible options on greater screen widths", () => {
 // . . .
```

This helps the test suite create human-readable error messages. I will merge as-is for this PR as this is not a huge deal- no need to change anything this time around.

Reply:
Thanks, I'll keep that in mind. 👍